WordPress/Classes/ViewRelated/Me/My Profile/Change Username/ChangeUsernameViewController.swift

```diff
@@ -55,12 +57,14 @@ private extension ChangeUsernameViewController {
         viewModel.suggestionsListener = { [weak self] state, suggestions, reloadAllSections in
             switch state {
             case .loading:
-                SVProgressHUD.show(withStatus: Constants.Alert.loading)
+                let cell = self?.tableView.cellForRow(at: IndexPath(row: 0, section: 1)) as? SearchTableViewCell
```

Comment:
We have multiple usages of this code all around, what do you think about extracting it to a method?

Reply:
Sure!